modules/light-clients/08-wasm/types/mock_engine_test.go

```diff
@@ -94,10 +116,14 @@ func (m *MockWasmEngine) Query(codeID wasmvm.Checksum, env wasmvmtypes.Env, quer
 
 // Sudo implements the WasmEngine interface.
 func (m *MockWasmEngine) Sudo(codeID wasmvm.Checksum, env wasmvmtypes.Env, sudoMsg []byte, store wasmvm.KVStore, goapi wasmvm.GoAPI, querier wasmvm.Querier, gasMeter wasmvm.GasMeter, gasLimit uint64, deserCost wasmvmtypes.UFraction) (*wasmvmtypes.Response, uint64, error) {
-	if m.SudoFn == nil {
-		panic("mock engine is not properly initialized")
+	msgTypeName := getQueryMsgPayloadTypeName(sudoMsg)
```

Comment:
Don't you need to create a new function called `getSudoMsgPayloadTypeName`? I think this function does not return the name of sudo messages

Reply:
whoops, good point here.